From d8a29c727d073b674adcf7ca122c3e5d04ba54ff Mon Sep 17 00:00:00 2001
From: Laura Abbott <labbott@redhat.com>
Date: Wed, 2 Jan 2019 08:13:21 -0800
Subject: [PATCH] Drop that def_bool

I don't know why

BUILDSTDERR: In file included from drivers/vfio/pci/trace.h:102,
BUILDSTDERR:                  from drivers/vfio/pci/vfio_pci_nvlink2.c:29:
BUILDSTDERR: ./include/trace/define_trace.h:89:42: fatal error: ./trace.h: No such file or directory
BUILDSTDERR:  #include TRACE_INCLUDE(TRACE_INCLUDE_FILE)
BUILDSTDERR:                                           ^
BUILDSTDERR: compilation terminated.
BUILDSTDERR: make[3]: *** [scripts/Makefile.build:277: drivers/vfio/pci/vfio_pci_nvlink2.o] Error 1
BUILDSTDERR: make[2]: *** [scripts/Makefile.build:492: drivers/vfio/pci] Error 2
BUILDSTDERR: make[1]: *** [scripts/Makefile.build:492: drivers/vfio] Error 2
BUILDSTDERR: make: *** [Makefile:1053: drivers] Error 2
BUILDSTDERR: make: *** Waiting for unfinished jobs....

Signed-off-by: Laura Abbott <labbott@redhat.com>
---
 drivers/vfio/pci/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/vfio/pci/Kconfig b/drivers/vfio/pci/Kconfig
index d0f8e4f5a039..11efd2099797 100644
--- a/drivers/vfio/pci/Kconfig
+++ b/drivers/vfio/pci/Kconfig
@@ -40,7 +40,7 @@ config VFIO_PCI_IGD
 	  To enable Intel IGD assignment through vfio-pci, say Y.
 
 config VFIO_PCI_NVLINK2
-	def_bool y
+	bool y
 	depends on VFIO_PCI && PPC_POWERNV
 	help
 	  VFIO PCI support for P9 Witherspoon machine with NVIDIA V100 GPUs
-- 
2.20.1

